spec/s3_meta_sync_spec.rb

```diff
@@ -388,6 +396,14 @@ def sh(command, options={})
           Uploading .s3-meta-sync
         TXT
       end
+
+      it "does download from remote with old .s3-meta-sync format if --no-local-changes set" do
+	sync("foo #{config[:bucket]}:bar #{params}")
+        `sed '/^:/d' ./foo/.s3-meta-sync > $TMPDIR/s3-tmp`
+        `mv $TMPDIR/s3-tmp ./foo/.s3-meta-sync`
+	result=`#{Bundler.root}/bin/s3-meta-sync #{config[:bucket]}:bar foo --no-local-changes 2>&1`
+	result.should_not =~ /NoMethodError/
```

Comment:
`.should_not include "NoMethodError"` reads a bit nicer and also produces a nicer error message